### DISCUSSION TITLE
Add ‘not interested’ button to Explore tab

### DISCUSSION THREAD

**Discussion by r2fo** [community/community#11238] [Mobile]
Allow us to say whether or not we are interested in repositories recommended by GitHub in the Explore Tab. I keep getting recommendations for repositories that I'm not interested in and there is no way to tell the AI to stop recommending me stuff related to that.

**Comment by brianlovin** [comment]
Thanks for the feedback @r2fo, I'll add this to our tracking issue for future Explore improvements.
